src/sounds.cpp

```diff
@@ -160,9 +161,48 @@ static std::vector<centroid> cluster_sounds( std::vector<std::pair<tripoint, int
     return sound_clusters;
 }
 
+int get_signal_for_hordes(int vol)
+{
+    int min_vol_for_hordes_cap = 61;
+    int per_level_down_vol_reduce_coef_for_hordes = 2;
+    int min_vol_for_hordes = 61;
+    int max_vol_for_hordes = 200;
+    int min_sig_power_for_hordes = 4;
+    int max_sig_power_for_hordes = 8;
+    //Lower the level- lower the sound
+    int vol_hordes=( (g->get_levz() < 0 ) ? vol/(per_level_down_vol_reduce_coef_for_hordes*std::abs(g->get_levz())) : vol);
+    //debugmsg( "vol_hordes %d  g->get_levz() %d ",vol_hordes, g->get_levz());
+    if( vol_hordes > min_vol_for_hordes_cap) {
+        vol_hordes = ( (vol_hordes > max_vol_for_hordes ) ? max_vol_for_hordes : vol_hordes);
+        // Formula for hordes hearing. Normalazing [min_vol_for_hordes,max_vol_for_hordes] ==> [min_sig_power_for_hordes,max_sig_power_for_hordes]
+        int sig_power = (vol_hordes-min_vol_for_hordes)*(max_sig_power_for_hordes-min_sig_power_for_hordes)/(max_vol_for_hordes-min_vol_for_hordes)+min_sig_power_for_hordes;
+        sig_power=( (sig_power <min_sig_power_for_hordes ) ? 0: min_sig_power_for_hordes);
+        //debugmsg( "vol %d  vol_hordes %d sig_power %d ", vol, vol_hordes, sig_power);
+        return sig_power;
+    }
+    return 0;
+}
+
+int get_signal_for_hordes_simple(int vol)
+{
+    int min_vol_for_hordes_cap = 61;
+    int per_level_down_vol_reduce_coef_for_hordes = 2;
+    int coef_for_hordes = 30;//= 1 overmap tile
+    //Lower the level- lower the sound
+    int vol_hordes=( (g->get_levz() < 0 ) ? vol/(per_level_down_vol_reduce_coef_for_hordes*std::abs(g->get_levz())) : vol);
+    //debugmsg( "vol_hordes %d  g->get_levz() %d ",vol_hordes, g->get_levz());
+    if( vol_hordes > min_vol_for_hordes_cap) {
+        //using simple formula
+        int sig_power =vol_hordes/coef_for_hordes +1;
+        //debugmsg( "vol %d  vol_hordes %d sig_power %d ", vol, vol_hordes, sig_power);
+        return sig_power;
+    }
+    return 0;
+}
+
 void sounds::process_sounds()
 {
-    std::vector<centroid> sound_clusters = cluster_sounds( recent_sounds );
+	std::vector<centroid> sound_clusters = cluster_sounds( recent_sounds );
```

Comment:
Looks like your IDE changed spaces here to tabs.
You'll need to change all tabs to 4 spaces each. Most IDEs support automatic tab->spaces.